### PR TITLE
fix: update scss to v2.13.2

### PR DESCRIPTION
**Pull request by zazzaz**

### CODE REVIEW

File: packages/theme-chalk/src/mixins/mixins.scss [hunk 1]
```diff
@@ -6,8 +6,7 @@
 @mixin res($key, $map: $--breakpoints) {
   // 循环断点Map，如果存在则返回
   @if map-has-key($map, $key) {
-    // use map in @media will cause an error with dart-sass, https://github.com/sass/dart-sass/issues/898
-    @media only screen and (inspect(map-get($map, $key))) {
```
Comment (zazzaz) [inline packages/theme-chalk/src/mixins/mixins.scss:10]: 可以保留

File: packages/theme-chalk/src/common/transition.scss [hunk 2]
```diff
@@ -4,8 +4,8 @@
 .fade-in-linear-leave-active {
   transition: $--fade-linear-transition;
 }
-.fade-in-linear-enter-from,
-.fade-in-linear-leave-from,
```
Comment (zazzaz) [inline packages/theme-chalk/src/common/transition.scss:8]: 可以保留

**Review by jw-foss — APPROVED** [review]

File: packages/theme-chalk/src/common/transition.scss [hunk 3]
```diff
@@ -24,31 +24,16 @@
 .el-fade-in-leave-active {
   transition: all .3s cubic-bezier(.55,0,.1,1);
 }
-.el-fade-in-enter-from,
```
Comment (SimonaliaChen) [inline packages/theme-chalk/src/common/transition.scss]: keep

File: packages/theme-chalk/src/common/transition.scss [hunk 4]
```diff
@@ -61,7 +46,7 @@
   transition: $--md-fade-transition;
   transform-origin: center top;
 }
-.el-zoom-in-top-enter-from,
+.el-zoom-in-top-enter,
```
Comment (SimonaliaChen) [inline packages/theme-chalk/src/common/transition.scss]: keep

File: packages/theme-chalk/src/common/transition.scss [hunk 5]
```diff
@@ -74,7 +59,7 @@
   transition: $--md-fade-transition;
   transform-origin: center bottom;
 }
-.el-zoom-in-bottom-enter-from,
+.el-zoom-in-bottom-enter,
```
Comment (SimonaliaChen) [inline packages/theme-chalk/src/common/transition.scss]: keep

File: packages/theme-chalk/src/common/transition.scss [hunk 6]
```diff
@@ -24,31 +24,16 @@
...
-.el-fade-in-top-leave-from{
-  opacity: 1;
-  transform: translate3d(0, 0, 0);
-}
-
 .el-zoom-in-center-enter-active,
 .el-zoom-in-center-leave-active {
   transition: all .3s cubic-bezier(.55,0,.1,1);
 }
-.el-zoom-in-center-enter-from,
```
Comment (SimonaliaChen) [inline packages/theme-chalk/src/common/transition.scss]: keep

File: packages/theme-chalk/src/common/transition.scss [hunk 7]
```diff
@@ -87,7 +72,7 @@
   transition: $--md-fade-transition;
   transform-origin: top left;
 }
-.el-zoom-in-left-enter-from,
```
Comment (SimonaliaChen) [inline packages/theme-chalk/src/common/transition.scss]: keep

File: packages/theme-chalk/src/common/transition.scss [hunk 8]
```diff
@@ -104,11 +89,11 @@
 .el-list-leave-active {
   transition: all 1s;
 }
-.el-list-enter-from, .el-list-leave-active {
```
Comment (SimonaliaChen) [inline packages/theme-chalk/src/common/transition.scss]: keep

**Review by SimonaliaChen — APPROVED** [review]

**Pull request merged**